packages/alea-frontend/pages/my-profile.tsx

```diff
@@ -138,6 +156,18 @@ const MyProfilePage: NextPage = () => {
         <h3 style={{ marginTop: '-15px' }}>
           <i>{userInfo.userId}</i>
         </h3>
+        {!isVerifiedUser && (
```

Comment:
Did you handle the case whereit is not email-passowrd login (IdM). In that case, we shouldn't show this.